tools/discovery/web/rev_proxy/HTTP-Traceroute.py

```diff
@@ -1,9 +1,16 @@
 #!/usr/bin/env python
```

Comment:
Better not do anything here. This has been added as a redistributed code from someone. Do not edit anything.